Mathlib/Data/Nat/Digits.lean

```diff
@@ -77,8 +81,8 @@ In any base, we have `ofDigits b L = L.foldr (fun x y ↦ x + b * y) 0`.
 * For `b = 1`, we define `digits 1 n = List.replicate n 1`.
 * For `b = 0`, we define `digits 0 n = [n]`, except `digits 0 0 = []`.
 
-Note this differs from the existing `Nat.to_digits` in core, which is used for printing numerals.
-In particular, `Nat.to_digits b 0 = [0]`, while `digits b 0 = []`.
+Note this differs from the existing `Nat.toDigits` in core, which is used for printing numerals.
+In particular, `Nat.toDigits b 0 = [0]`, while `digits b 0 = []`.
```

Comment:
I realise that this was already like this, but another important difference is that `Nat.toDigits` returns a list of *characters*, while `Nat.digits` yields a list of `Nat`!